db/schema.rb

```diff
@@ -68,6 +68,7 @@
     t.datetime "updated_at"
     t.integer  "course_id",          limit: 4
     t.string   "display_name",       limit: 255
+    t.integer  "category_id",        limit: 4
```

Comment:
Migrations are trying to scope cud for `has_joined` and can't find the column now...